src/rules/declaration-block-no-redundant-longhand-properties/README.md

```diff
@@ -0,0 +1,102 @@
+# declaration-block-no-redundant-longhand-properties
+
+Disallow longhand properties that can be combined into one.
+
+```css
+a { padding-top: 1px; padding-right: 1px; padding-bottom: 1px; padding-left: 1px }
+/**                                     ↑
+ *                            This longhand properties */
```

Comment:
`These longhand properties`, i.e use the _pluaral_ `these` rather than _singular_ `this`